test/functional/normalization/json_parsing_test.go

```diff
@@ -0,0 +1,177 @@
+package normalization
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+)
+
+const (
+	Json = `
+    {
+      "a": "Alpha",
+      "b": true,
+      "c": 12345,
+      "d": [
+      true,
+      [
+        false,
+        [
+        -123456789,
+        null
+        ],
+        3.9676,
+        [
+        "Something else.",
+        false
+        ],
+        null
+      ]
+      ],
+      "e": {
+      "zero": null,
+      "one": 1,
+      "two": 2,
+      "three": [
+        3
+      ],
+      "four": [
+        0,
+        1,
+        2,
+        3,
+        4
+      ]
+      },
+      "f": null,
+      "h": {
+        "a": {
+          "b": {
+            "c": {
+              "d": {
+                "e": {
+                  "f": {
+                    "g": null
+                     }
+                   }
+                 }
+               }
+             }
+           }
+         },
+      "i": [
+            [
+             [
+              [
+               [
+                [
+                 [
+                  null
+                 ]
+                ]
+               ]
+              ]
+             ]
+            ]
+          ]
+    }
+    `
+)
```

Comment:
Wooohooo, now *thats* a test case! :+1:

Reply:
copied it from https://github.com/anilmaurya/fast_jsonparser/blob/master/benchmark/json_benchmark.rb#L3